ios/RNTPTDocumentView.m

```diff
@@ -127,6 +127,48 @@ - (nullable PTToolManager *)toolManager
     return self.documentViewController.toolManager;
 }
 
+#pragma mark - Document Openining
+
+-(void)openDocument
+{
```

Comment:
Should return here without doing anything if `documentViewController` is nil

Reply:
Done! Thanks.